components/script/dom/gpuqueue.rs

```diff
@@ -192,4 +196,40 @@ impl GPUQueueMethods for GPUQueue {
 
         Ok(())
     }
+
+    /// https://gpuweb.github.io/gpuweb/#dom-gpuqueue-onsubmittedworkdone
```

Comment:
To avoid a rustdoc warning:

```suggestion
    /// <https://gpuweb.github.io/gpuweb/#dom-gpuqueue-onsubmittedworkdone>
```